docs/index.md

```diff
@@ -17,27 +27,43 @@
 
 ```
      <dependency>
-        <groupId>org.wso2.extension.siddhi.io.</groupId>
-        <artifactId>siddhi-io-</artifactId>
+        <groupId>org.wso2.extension.siddhi.io.hl7</groupId>
+        <artifactId>siddhi-io-hl7</artifactId>
         <version>x.x.x</version>
      </dependency>
 ```
 
+## Jenkins Build Status
+
```

Comment:
Please update the build status as in https://github.com/wso2-extensions/siddhi-store-elasticsearch/blame/master/README.md#L55